scout/server/blueprints/cases/templates/cases/case_report.html

```diff
@@ -857,240 +868,24 @@ <h4>Scout case analysis report</h4> - created on:&nbsp;<strong>{{report_created_
 </div>
 {% endmacro %}
 
-{% macro sv_variant_content(variant,index) %} <!--The entire description of a structural variant, dynamic contentc -->
```

Comment:
Eh, what happened there?!

Reply:
If you open master you'll see that there are 2 sv_variant_content macros on the same page. Of course only the last one was used in the end. I must have forgotten one when I touched this code last!

Reply:
Ok! Looks good when looking at the whole file, but really weird on the diff! :blush: